src/py/pyproject.toml

```
@@ -1,3 +1,33 @@
 [build-system]
-requires = ["setuptools>=42"]
+requires = ["setuptools>=61.2"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "pyodide-py"
+version = "0.26.0.dev0"
+authors = [{name = "Pyodide developers"}]
+description = "A Python package providing core interpreter functionality for Pyodide"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
+    "Operating System :: OS Independent",
+]
+requires-python = ">=3.8"
```

This is quite out of date... Probably this would be closer to accurate:
```suggestion
requires-python = ">=3.11"
```